deprecated-dune-v1-abstractions/ethereum/dex/trades/insert_1inch.sql

```diff
@@ -283,6 +289,18 @@ WITH rows AS (
             where call_success
                 and call_block_time >= start_ts
                 and call_block_time < end_ts
+            -- union all
+            -- select "output_returnAmount", "amount", "srcToken", "pools", "call_tx_hash", "call_trace_address", "call_block_time", "contract_address"
```

Comment:
the ```pools``` column in both of these v5 decoded tables are numeric data type, while prior tables in the union are bytea, causing failures and unable to load. i've commented these out and continued loading

Reply:
Is this the reason why 1inch isn't in `dex.trades` currently?

Reply:
yes, it's reprocessing history with changes here (it takes a long time)